components/frontend/src/header_footer/buttons/ReportPeriodButton.jsx

```diff
@@ -0,0 +1,30 @@
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown"
+import ArrowRightIcon from "@mui/icons-material/ArrowRight"
+import EventIcon from "@mui/icons-material/Event"
+import { Stack } from "@mui/material"
+import { bool, func } from "prop-types"
+
+import { datePropType } from "../../sharedPropTypes"
+import { AppBarButton } from "./AppBarbutton"
+
+export function ReportPeriodButton({ reportDate, setSettingsPanelVisible, settingsPanelVisible }) {
+    return (
+        <AppBarButton
+            tooltip="Change report date, number of dates shown, time between dates, and date order"
+            startIcon={
+                <Stack direction="row" spacing={0}>
+                    {settingsPanelVisible ? <ArrowDropDownIcon /> : <ArrowRightIcon />}
+                    <EventIcon />
+                </Stack>
+            }
+            onClick={() => setSettingsPanelVisible(!settingsPanelVisible)}
```

Comment:
How does the click know that it's opening the period panel and not the settings panel; shouldn't it also pass `visibleSettingsPanel` ?